examples/cifar10/pt/utils/cifar10_data_splitter.py

```diff
@@ -61,12 +61,13 @@ def _get_site_class_summary(train_label, site_idx):
 
 
 class Cifar10DataSplitter(FLComponent):
-    def __init__(self, split_dir: str = None, num_sites: int = 8, alpha: float = 0.5, seed: int = 0):
+    def __init__(self, split_dir: str = None, num_sites: int = 8, alpha: float = 0.5, seed: int = 0, custom_CIFAR10_ROOT: str = None):
```

Comment:
suggest passing in "cifar_root", or "data_root"

then set "self.cifar_root" or "self.data_root"